packages/vscode/src/extension.ts

```diff
@@ -0,0 +1,20 @@
+import * as vscode from "vscode";
+import { StatusBarItem } from "./statusBarItem";
+
+export function activate(context: vscode.ExtensionContext) {
+  console.log("Starting Coauthors ...");
+
+  context.subscriptions.push(
+    vscode.commands.registerCommand("coauthors.add", () => {
```

Comment:
Simple question) I see you are doing "something" via the coauthors.add command, but where is the script for "something" specified?

Reply:
or is it simply initialization, adding code that is later executed by this command ?